### ISSUE TITLE
"CredentialStatusList2017" treated as invalid/unknown element in JSON-LD normalization.

### ISSUE THREAD

**Issue by sudeshrshetty** [hyperledger/aries-framework-go#1697]

**Comment by troyronda** [comment]
`CredentialStatusList2017` should be treated as an example. Need to use an example context.

**Comment by kdimak** [comment]
@troyronda added https://trustbloc.github.io/context/vc/examples-v1.jsonld

**Comment by troyronda** [comment]
@kdimak yup .... I was curious if we can cache a context locally in the repo in case the above link goes down or disappears. (to make tests less fragile).

**Comment by kdimak** [comment]
yes, this makes sense @troyronda 
I created https://github.com/hyperledger/aries-framework-go/issues/1753.
I think we can close this issue as to close https://github.com/hyperledger/aries-framework-go/pull/1735 we added `WithExternalJSONLDContext()` `WithJSONLDOnlyValidRDF()` options.

**Comment by troyronda** [comment]
@sudeshrshetty can this be closed?
